server/helpers/mailer.js

```diff
@@ -0,0 +1,78 @@
+import sendgridMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+import debug from 'debug';
+
+dotenv.config();
+
+const log = debug('dev');
+
+sendgridMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/**
+ * Mailer Class
+ */
+class Mailer {
+  /**
+   * Email trasporter
+   * @param {String} to - Reciever email
+   * @param {String} from - Sender email
+   * @param {String} subject - Email subject
+   * @param {String} html - Email body
+   * @returns  {Object} - Mailer response
+   */
+  static sendMail(to, from, subject, html) {
```

Comment:
We're supposed to convert this to async/await method.
Then we can convert the `.then` to `try/catch`